torch/jit/__init__.py

```diff
@@ -688,12 +688,32 @@ def trace(func,
     Trace a function and return an executable ``ScriptModule`` or ``torch._C.Function``
     that will be optimized using just-in-time compilation.
 
-    .. warning::
+    Using ``torch.jit.trace`` and :func:`torch.jit.trace_module<torch.jit.trace_module>`, you can turn an existing module or Python
+    function into a TorchScript ``torch._C.Function`` or ``ScriptModule``. You must provide example inputs,
```

Comment:
Please do not document _C as a public facing API. We should make the functions torch.jit.Function